source/patterns/@aws-solutions-constructs/aws-eventbridge-sns/test/eventbridge-sns-topic.test.ts

```diff
@@ -213,63 +213,6 @@ test('check the sns topic properties with existing KMS key', () => {
   });
 
   expect(stack).toHaveResource('AWS::KMS::Key', {
-    KeyPolicy: {
```

Comment:
This seems to no longer check a good deal of stuff for correctness...why is this?

Reply:
It is asserting that customer generated CMK is used instead of the AWS managed KMS, setting this [this](https://github.com/awslabs/aws-solutions-constructs/blob/fix-unit-test-kms/source/patterns/%40aws-solutions-constructs/aws-eventbridge-sns/test/eventbridge-sns-topic.test.ts#L194) has to produce [this](https://github.com/awslabs/aws-solutions-constructs/blob/fix-unit-test-kms/source/patterns/%40aws-solutions-constructs/aws-eventbridge-sns/test/eventbridge-sns-topic.test.ts#L216) expected result